packages/frontend/src/components/LoginForm.tsx

```diff
@@ -389,21 +410,33 @@ export function ConfigureProgressDialog({
       ;(async () => {
         try {
           // Prepare initial configuration
-          const initialConfig: { [key: string]: string } = {
+          const initialConfig: Credentials = {
             ...credentials,
-            verified_one_on_one_chats: '1',
           }
-          await BackendRemote.rpc.batchSetConfig(accountId, initialConfig)
+          const { proxyEnabled, proxyUrl, ...transportConfig } = initialConfig
+          // Set proxy settings if changed!
+          if (proxyUpdated) {
+            await BackendRemote.rpc.batchSetConfig(accountId, {
+              proxy_enabled:
+                proxyEnabled === true ? Proxy.ENABLED : Proxy.DISABLED,
+              proxy_url: proxyUrl,
+              verified_one_on_one_chats: '1', // do we still need this?
```

Comment:
@Simon-Laux what is th reason we have to configure a value that is always "hard coded" and can't be changed in the UI?